recipes/tk/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = "8.5.19" %}
+{% set maj_min = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: tk
+  version: {{ version }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - python         # [win]
+    - curl           # [win]
+    - 7za            # [win]
+
+test:
+  requires:
+    - python 2.7.*   # [win and py27]
+    - python 3.4.*   # [win and py34]
+    - python 3.5.*   # [win and py35]
+
+  files:
+    - hello.tcl
+
+  commands:
+    # Check for binaries.
+    {% for ver_suffix in ["", maj_min] %}
+    - test -f "${PREFIX}/bin/tclsh{{ ver_suffix }}"                           # [unix]
+    - test -f "${PREFIX}/bin/wish{{ ver_suffix }}"                            # [unix]
+    - if not exist %LIBRARY_PREFIX%\\bin\\tclsh{{ ver_suffix }}.exe exit 1    # [win]
```

Comment:
I don't think we include it in defaults.  Probably the thinking is that tcl/tk are used solely as libraries.  I'm not sure I totally agree with that thinking.  Easy to include it here, in case anyone ever needs it.  The downside of that is that we're dumping more and more files into Library\bin.  Doesn't bother me, but other people don't like it.